CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## v1.0.0-rc3 (unreleased)
 
-- No changes yet.
+- **[Breaking]** Rename `fx.Inject` to `fx.Extract` to avoid using the
+  term "inject" outside of the ideal usecase. `fx.Extract` is a hammer to
+  break glass, not a way to achieve pure dependency injection.
```

Comment:
Don't think we should include justifications for change in a changelog, as they can get pretty wordy. We should just summarize the changes right?

Or what's the policy here